meta/main.yml

```diff
@@ -1,7 +1,4 @@
 ---
-# Copyright (C) 2020 NSPCC <https://nspcc.ru/>
-# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Why?

Reply:
SPDX makes things easier to process by various tools, removing it doesn't help anyone.